### ISSUE TITLE
Shui для FBG5 проблема с WIFI на ESP8266

### ISSUE THREAD

**Issue by Alex7294** [vyacheslav-shubin/shui#15]
Добрый день! После перепрошивки FBG5 на шуи принтер перестал принимать настройки печати g-code по вифи , т.е. (передача g-code идет, но после окончания передачи на главном экране печати остается лишь температура стола и сопла на данный момент 26 градусов , а температура при которой должен печатать сей агрегат стоит на 0) 
как же всетаки исправить данную проблему?! (перепрошивал по новой, менял слайсеры, через флешку сохраненный g-code печатает)

**Comment by vyacheslav-shubin** [comment]
https://t.me/+Yoo5wEKo7okyMGQy

**Comment by RAFALFRY** [comment]
Hi,
Did you manage to solve the problem with printing via WiFi?
My printer with the MKS PWC module resets after the file upload is completed.

**Comment by Alex7294** [comment]
> Hi, Did you manage to solve the problem with printing via WiFi? My printer with the MKS PWC module resets after the file upload is completed.

Yes, everything worked out... it turned out that shui does not support fat32 for the drive, although the standard firmware was sitting on this format! transfer the drive to ntfs and you will be happy)))

**Comment by RAFALFRY** [comment]
Thanks.
For me, formatting from FAT to FAT32 helped.
I will add that in the older version of SHUI the FAT format did not bother me.
